src/components/Dashboard/Balance.js

```diff
@@ -218,7 +218,9 @@ const LatestActivity = ({ activity, distribution }) => {
             color: ${theme.content};
           `}
         >
-          {convertToString(activity.type, activity.direction)}
+          {activity.isImmediate && !activity.isEffective
+            ? 'Effective next term'
+            : convertToString(activity.type, activity.direction)}
```

Comment:
I'm wondering if we should have a tooltip for the `Active` part of the balance module when it's `Effective next term`.

I can still see a user being confused about their "immediate" active balance vs. their "effective" active balance at the moment, as we only show one value in the module.